bin/wazo-dist-upgrade-bullseye-base

```diff
@@ -74,9 +74,25 @@ change_sources_list() {
     from=$1
     to=$2
     if [ ${from} == 'buster' ] && [ ${to} == 'bullseye' ]; then
-        sed -i "s|\(security.*\) buster/updates|\1 bullseye-security|" /etc/apt/sources.list
+        sed -i "/\(security.*\) buster.*/d" /etc/apt/sources.list
+        sed -i "/\(archive.*\) buster.*/d" /etc/apt/sources.list
```

Comment:
can't we simplify this to `/ buster.*/d`?

Reply:
I guess, but it works flawless for the last 2 years... :crossed_fingers: